views/partials/applicationModal.hbs

```diff
@@ -5,10 +5,10 @@
 
         <!-- CONTACT INFO -->
         <p class='applicant-contact'>
-            <div class="contact-wrapper">
-                <div class="contact-item-wrapper"><span class='fa fa-envelope-o'></span><span name='email'></span></div>
-                <div class="contact-item-wrapper"><span class='fa fa-mobile'></span><span name='phone'></span></div>
-            </div>
+        <div class="contact-wrapper">
+            <div class="contact-item-wrapper"><span class='fa fa-envelope-o'></span><span name='email'></span></div>
+            <div class="contact-item-wrapper"><span class='fa fa-mobile'></span><span name='phone'></span></div>
+        </div>
```

Comment:
Yeah I do, I didn't realize that Husky ran prettier *after* the push, I thought it would run and amend the commit with the prettified version. Anyways I was able to get the build passing with that, so thanks!